doc/btp_bas.txt

```diff
@@ -0,0 +1,183 @@
+BAS Service (ID 30)
+=====================
+
+Commands and responses:
+
+		Opcode 0x00 - Error response
+
+		Opcode 0x01 - Read Supported Commands command/response
+
+				Controller Index:	<controller id>
+				Command parameters:	<none>
+				Response parameters:	<supported commands> (variable)
+
+				Each bit in response is a flag indicating if command with
+				opcode matching bit number is supported. Bit set to 1 means
+				that command is supported. Bit 0 is reserved and shall always
+				be set to 0. If specific bit is not present in response (less
+				than required bytes received) it shall be assumed that command
+				is not supported.
+
+				In case of an error, the error response will be returned.
+
+		Opcode 0x02 - BAS Set Battery Level
+
+				Controller Index:		<controller id>
+				Command parameters:		Address_Type (1 octet)
+										Address (6 octets)
+										Battery Level (1 octet)
+				Response parameters:	<None>
+
+				This command instruct the IUT to change the Battery Level field of 
+				the Battery Level Status characteristic.
+
+		Opcode 0x03 - BAS Set Battery Present
+
+				Controller Index:		<controller id>
+				Command parameters:		Address_Type (1 octet)
+										Address (6 octets)
+										Battery Present (1 octet)
+				Response parameters:	<None>
+
+				This command instruct the IUT to change the Power State field of 
+				the Battery Level Status characteristic.
+
+				BTP Values:
+						BTP_BAS_BLS_BATTERY_NOT_PRESENT = 0,
+						BTP_BAS_BLS_BATTERY_PRESENT = 1
```

Comment:
I think the format should be more like
```suggestion
				Battery Present:
					0 = Battery not present
					1 = Baterry present
```
or
```suggestion
				Battery Present:
					0	Battery not present
					1	Baterry present
```

To match most other commands. We don't usually refer to the values defined in Python (but rather use text), and the value always seems to come first (but there are quite a few different formats)

This applies to most other these commands